resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -8,28 +8,48 @@
 
 parser = argparse.ArgumentParser(description="Create igv session.")
 
-parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV")
+parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV.")
 
 args = parser.parse_args()
 
 
+def get_build_info(input_file):
+    """Get build information in UCSC notation, if possible."""
+    build_dict = {
+        'hg38': ['GRCh38'],
+        'hg19': ['GRCh37', 'b37'],
+        'mm10': ['GRCm38'],
+        'mm9': ['MGSCv37'],
+        'rn6': ['Rnor_6.0'], }
+
+    _, build, _, _ = input_file.split('_')
```

Comment:
This line silently fails in many data objects

E.g. data object on qa2 finished successfully [qa2/data/sample-archive-13](https://qa2.genialis.com/universe/bioinformatics/data/sample-archive-13) even though this line raised an error [qa2/data/sample-archive-13/stdout.txt](https://qa2.genialis.com/universe/bioinformatics/data/sample-archive-13/file/stdout.txt).
And some on app [app/data/sample-archive-46/stdout.txt](https://app.genialis.com/universe/bioinformatics/data/sample-archive-46/file/stdout.txt?_b=), [app/data/sample-archive-43/stdout.txt](https://app.genialis.com/universe/bioinformatics/data/sample-archive-43/file/stdout.txt?_b=).

To make the error non-silent, you can add `re-checkrc` after calling `make_igv_session_archive.py`, and you can prevent the error by checking if file looks to be in the correct format